### PR TITLE
🌱 Transport cmd

### PR DESCRIPTION
**Pull request by nirrozenbaum**
## Related issue(s)

Fixes #

### CODE REVIEW

**Review by shmuelk — APPROVED** [review]
/lgtm

File: cmd/transport/base-controller.go [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+	go func() {
+		err := http.ListenAndServe(serverBindAddress, mymux)
+		if err != nil {
+			logger.Error(err, "Failure in web serving")
+			klog.FlushAndExit(klog.ExitFlushTimeout, 1)
+		}
+	}()
+}
+
+func Run(transportImplementation transport.Transport) {
```
Comment (vMaroon) [inline cmd/transport/base-controller.go:89]: Who calls this? I think the step of actual execution should be a part of this PR. If gotta merge due to schedule, understandable.

**Review by vMaroon — APPROVED** [review]

**Pull request merged**